resources/public/api/conf/1.0/docs/errors.md

```diff
@@ -27,3 +27,21 @@ Where possible, multiple errors will be returned with `INVALID_REQUEST` in the f
         }
     ]
 }`
+
+Where `FORMAT_VALUE` errors are returned, `message` will describe the expected format and `paths` will show which fields are invalid, in the following format:<br>
+  `{
+      "code": "INVALID_REQUEST",
+      "message": "Invalid request",
+      "errors": [
+          {
+             "code": "FORMAT_VALUE",
+             "message": "The field should be a positive value greater than 0 and no greater than 99999999999.99",
+             "paths": ["/objectName/fieldName1", "/objectName/fieldName2"]
+          },
+          {
+             "code": "FORMAT_VALUE",
+             "message": "The field should be between 0 and 99999999999.99",
+             "paths": ["/objectName/fieldName3", "/objectName/fieldName4"]
+          }
+      ]
+  }`
```

Comment:
When looking at the front end for RAML it looks off with weird lines.

May want to check on that, or ask about a better way of implementation to Mashuk, Nick and Casey